aksel.nav.no/website/pages/produktbloggen/generateRssFeed.tsx

```diff
@@ -0,0 +1,27 @@
+import RSS from "rss";
+import fs from "fs";
+
+export default async function generateRssFeed(bloggposts) {
+  const site_url = "https://aksel.nav.no/";
+  const path = "produktbloggen";
```

Comment:
Er det mulig å gjøre denne funksjonen mer fleksibel (så vi kan ha RSS-feed for de andre artiklene også) ved å sende path inn som et parameter til funksjonen?

Reply:
Ja det var en god idee. Jeg har ikke oversikt over hvilke andre steder dette kunne trenges men jeg kan endre koden til å ta imot en `path`.